Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM ubuntu:18.04
 
+ENV DEBIAN_FRONTEND=noninteractive
+
 COPY forge /bin
 
 RUN apt-get -y update && apt-get -y install ca-certificates
+RUN useradd -ms /bin/bash forge
+USER forge
```

Comment:
Does this apply to images build on this one as well? Se https://github.com/stormforger/gulp-template/blob/master/Dockerfile e.g.

Reply:
Since that only copies the binary it would be run there as `root` again, I guess. We would need to apply the patch there too.

Reply:
Ok. Since we have no data, that shouldn't be a problem.